postprocessing/land_utils/is-compressed.c

```diff
@@ -85,18 +85,19 @@ int main(int argc, char* argv[])
 
    int optind = argc - 1;
    
-   __NC_ASRT__(nc_open(argv[optind],NC_NOWRITE,&ncid));
-   __NC_ASRT__(nc_inq_ndims(ncid,&ndims));
+   CHECK_NC_ERRSTAT(nc_open(argv[optind],NC_NOWRITE,&ncid));
+   CHECK_NC_ERRSTAT(nc_inq_ndims(ncid,&ndims));
    if (verbose) printf("Found %d dimensions\n",ndims);
    for (dimid=0; dimid<ndims; dimid++) {
-      __NC_ASRT__(nc_inq_dimname(ncid,dimid,name));
+      CHECK_NC_ERRSTAT(nc_inq_dimname(ncid,dimid,name));
       if (verbose) printf("Examining \"%s\"\n",name);
       if (nc_inq_varid(ncid,name,&varid)==NC_NOERR &&
           nc_inq_attid(ncid,varid,"compress",&attid)==NC_NOERR) {
           return 0;
       }
    }
-   __NC_ASRT__(nc_close(ncid));
+   CHECK_NC_ERRSTAT(nc_sync(ncid));
```

Comment:
With the `NC_NOWRITE` option to the open, this application will not be modifying the file and shouldn't need to sync prior to closing.